### ISSUE TITLE
PDFSyntaxError('No /Root object! - Is this really a PDF?')

### ISSUE THREAD

**Issue by jlegaye** [euske/pdfminer#98]
I have a PDF file containing only a picture. I can open this PDF without difficulty.
When I run this command ont this PDF :
```
pdf2txt.py -S -t xml -o pdfMinerOutput.xml input.pdf
```

I get this stack trace :
```
Traceback (most recent call last):
  File "C:\Tools\Python27\Scripts\pdf2txt.py", line 116, in <module>
    if __name__ == '__main__': sys.exit(main(sys.argv))
  File "C:\Tools\Python27\Scripts\pdf2txt.py", line 108, in main
    caching=caching, check_extractable=True):
  File "C:\Tools\Python27\lib\site-packages\pdfminer\pdfpage.py", line 122, in g
et_pages
    doc = PDFDocument(parser, password=password, caching=caching)
  File "C:\Tools\Python27\lib\site-packages\pdfminer\pdfdocument.py", line 583,
in __init__
    raise PDFSyntaxError('No /Root object! - Is this really a PDF?')
pdfminer.pdfparser.PDFSyntaxError: No /Root object! - Is this really a PDF?
```

**Comment by PJaros** [comment]
TL;DR One of my Files edited with PDFXChangeViewer 2.0 (Build 42.1) opening with "pdfminer 20140328" will result in "pdfminer.pdfparser.PDFSyntaxError: No /Root object! - Is this really a PDF?" 

I have the same error. I'm using Ubuntu 14.04 32-bit. I create my PDF-files tesseract 3.03.02-3 and they can be opend with pdfminer without problem. Editing them with PDFXChangeViewer 2.0 (Build 42.1) will render them un-openable. I'm using pdfminer calling from my own programm. I can provide test PDFs and sample-code if needed.

**Comment by zixuzhang** [comment]
I have the same error too
